gio/src/auto/native_volume_monitor.rs

```diff
@@ -0,0 +1,27 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::VolumeMonitor;
+use std::fmt;
+
+glib::wrapper! {
+    #[doc(alias = "GNativeVolumeMonitor")]
+    pub struct NativeVolumeMonitor(Object<ffi::GNativeVolumeMonitor, ffi::GNativeVolumeMonitorClass>) @extends VolumeMonitor;
```

Comment:
This probably shouldn't have been in the .gir file. AFAIU it's private and exposed only via the parent class